From cb94c82514159f2f576238d7b64a96fef3bd7ff5 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:04:50 -0400
Subject: [redhat] kabi: Adding symbol hrtimer_forward (kernel/hrtimer.c)

Message-id: <adac4bb6ef635e5bccc02660475eda3c025641f6.1412771859.git.poros@redhat.com>
Patchwork-id: 97186
O-Subject: [RHEL7.1 PATCH 30/67] kabi: Adding symbol hrtimer_forward (kernel/hrtimer.c)
Bugzilla: 1148054
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148054
    https://bugzilla.redhat.com/show_bug.cgi?id=1148054

    Description:
    ============
    Called indirectly from hr_timer_forward_now(). move SGi hi-res timer expiry times
    Symbol used in extint_timer.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/hrtimer_forward
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/hrtimer_forward
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/hrtimer_forward

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_forward b/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_forward
new file mode 100644
index 0000000..a640de2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/hrtimer_forward
@@ -0,0 +1,2 @@
+#1-
+0xa7a4705c hrtimer_forward vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/hrtimer_forward b/redhat/kabi/kabi-module/kabi_s390x/hrtimer_forward
new file mode 100644
index 0000000..6326503
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/hrtimer_forward
@@ -0,0 +1,2 @@
+#1-
+0xde6fd51f hrtimer_forward vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_forward b/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_forward
new file mode 100644
index 0000000..daee7df
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/hrtimer_forward
@@ -0,0 +1,2 @@
+#1-
+0xe7f3608a hrtimer_forward vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1